src/wwwroot/services/settings.js

```diff
@@ -137,5 +138,18 @@
 
       return plansCache;
     }
+
+    function getCountries() {
```

Comment:
Yes in the `signup `service, do you think about injecting `signup `service on the `billing `controller in use it?